models.js

```diff
@@ -1,8 +1,34 @@
-const db = require('./db/connection')
+const db = require('./db/connection');
+const articles = require('./db/data/test-data/articles');
 
 exports.getTopicModel = () => {
     return db.query('SELECT * FROM topics;')
         .then( (topics) => {
-            return topics.rows
+            return topics.rows;
         })
-}
\ No newline at end of file
+}
+
+exports.getArticleModel = () => {
+    return Promise.all([
+        db.query('SELECT * FROM articles ORDER BY articles.created_at DESC;')
+        .then( (articles) => {
+            return articles.rows;
+        }),
+        db.query('SELECT article_id FROM comments;')
+        .then( (articles) => {
+            return articles.rows;
+        })
+    ])
+    .then( (promiseArr) => {
+        promiseArr[0].forEach(article => {
+            article.comment_count = 0;
+            promiseArr[1].forEach(comment => {
+                if (article.article_id === comment.article_id) {

```

Comment:
it's impressive that you've managed to achieve a comment count through javascript - but this is more easily achievable by just using SQL making use of `COUNT` and `JOIN` functionality 

I would say that you're missing an opportunity here to showcase SQL skills to employers, and so I would recommend refactoring this to just use one single SQL query that joins and creates a `comment_count` with the `COUNT` function